include/git2/repository.h

```diff
@@ -640,6 +640,18 @@ GIT_EXTERN(int) git_repository_message_remove(git_repository *repo);
  */
 GIT_EXTERN(int) git_repository_state_cleanup(git_repository *repo);
 
+/**
+ * Callback used to iterate over each FETCH_HEAD entries
```

Comment:
Typo: "entry"

Reply:
Isn't `FETCH_HEAD` a file whose contents a list of "one or more entries" ? I find it grammatically confusing that you can singularize over "one or more entries"… I can rephrase as "Callback used to iterate over FETCH_HEAD contents`, but that's still plural 🤣.

Reply:
It is. But you're not iterating over each entries, but over each entry :)